scripts/backfill_genome_data/sequence_read_sets.py

```diff
@@ -0,0 +1,48 @@
+"""
+Find all fastq.gz files within a provided directory and group them by samples
+into sequence read sets.
+
+All sequence read sets are output to stdout as newline-delimited JSON
+records.  You will likely want to redirect stdout to a file.
+"""
+import glob
+import json
+import argparse
+
+def find_fasta_urls(fasta_directory) -> dict:
+    """
+    Find all fastq.gz files within provided *fastq_directory*.
+
+    Return as a dict with keys as NWGC sample IDs and values as an array of
+    file paths.
+    """
+    sequence_read_sets = {}
+    for filename in glob.glob(fasta_directory + "/*.fastq.gz"):
+        sample = filename.split('/')[-1].split('_')[0]
+        if not sequence_read_sets.get(sample):
+            sequence_read_sets[sample] = [filename]
+        else:
+            sequence_read_sets[sample].append(filename)
+    return sequence_read_sets
+
+
+def dump_ndjson(sequence_read_set_dict):
+    """
+    Prints sequence read sets as newline-delimited JSON records.
+    """
+    for sample in sequence_read_set_dict:
+        print(json.dumps({"sample": sample, "urls": sequence_read_set_dict[sample]}))
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        description = __doc__,
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter
+    )
+    parser.add_argument("directory", type=str, nargs='?',
```

Comment:
Why `nargs='?'`? I think that's only useful when there's also a `default` or `const` value.

Ditto for the other scripts in this PR.